.circleci/config.yml

```diff
@@ -738,6 +751,57 @@ commands:
       - store_artifacts:
           path: << parameters.artifact_name >>.zip
 
+  store-firebase-artifacts:
+    parameters:
+      artifact_name:
+        type: string
+        default: "firebase"
+      derived_data_path:
+        type: string
+        default: "build"
+    description: "Save firebase Results as Artifacts"
+    steps:
+      - run:
+          name: Zipping firebase results
+          command: |
+            export ZIP_FILE="$PWD/<< parameters.artifact_name >>.zip"
+            cd "<< parameters.derived_data_path >>"
+            zip -r "$ZIP_FILE" xcodebuild_output.log || echo "Skipping xcodebuild_output.log"
+            zip -r "$ZIP_FILE" video.mp4 || echo "Skipping video.mp4"
+            zip -r "$ZIP_FILE" test_result_0.xml || echo "test_result_0.xml"
+            zip -r "$ZIP_FILE" syslog.txt || echo "Skipping syslog.txt"
+            zip -r "$ZIP_FILE" TestLogs || echo "Skipping TestLogs"
+            zip -r "$ZIP_FILE" ProfileData || echo "Skipping ProfileData"
+            zip -r "$ZIP_FILE" CrashLogs || echo "Skipping CrashLogs"
+
+          when: always
+      - store_artifacts:
+          path: << parameters.artifact_name >>.zip
+
+  install-gcloud:
+    description: "Install gcloud"
+    steps:
+      - run:
+          name: Install gcloud
+          command: |
+            brew install ccache
+            mkdir /tmp/gcloud && cd /tmp/gcloud
+            curl -OL https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-366.0.0-darwin-x86_64.tar.gz
+            echo '7241cf19d0116f82d4600c2e60c7f63f084c384759b1febabaafe11501ae81fe *google-cloud-sdk-366.0.0-darwin-x86_64.tar.gz' >> checksumfile
```

Comment:
Can we extract google-cloud-sdk version and checksum as command parameters with default values? It would be easier to update them later.

Reply:
Can we also leave a link to the documentation where we can find hashsum for a newer version?